sql/mysql/slashschema_create.sql

```diff
@@ -1491,7 +1491,7 @@ CREATE TABLE users_comments_read_log (
 	uid MEDIUMINT UNSIGNED NOT NULL,
 	discussion_id MEDIUMINT UNSIGNED NOT NULL,
 	cid INT UNSIGNED NOT NULL,
-	UNIQUE (discussion_id,uid,cid)
+	UNIQUE KEY didnuid (discussion_id,uid)
```

Comment:
Please match this to what is in MySQL updates, or just let the updates fix it, that is why we have the automagic system.